package.json

```diff
@@ -63,6 +63,7 @@
     "@babel/core": "^7.20.0",
     "@babel/preset-env": "^7.20.0",
     "@types/jest": "^29.5.11",
+    "jest": "^29.5.11",
```

Comment:
Had to add `jest` cause we're using it now in `jest-mocks.ts`

Reply:
Sure, but it's not used at runtime by the module. Having only the type is enough: it will fails if the user importing it does not have the `jest` dependency only.